dot/rpc/modules/state.go

```diff
@@ -489,6 +495,41 @@ func (sm *StateModule) QueryStorage(
 	return nil
 }
 
+// QueryStorageAt queries historical storage entries (by key) at the best block if the given block hash is nil
+func (sm *StateModule) QueryStorageAt(
+	_ *http.Request, req *StateStorageQueryAtRequest, res *[]StorageChangeSetResponse) error {
```

Comment:
nit `req` -> `request` and `res` -> `response` (see comment below to avoid name conflict)